framework/tlvf/AutoGenerated/src/CmduMessageRx.cpp

```diff
@@ -49,7 +250,17 @@ std::shared_ptr<cCmduHeader> CmduMessageRx::parse(uint8_t *buff, size_t buff_len
     m_cmdu_header = std::make_shared<cCmduHeader>(buff, buff_len, true, false);
     if (!m_cmdu_header || m_cmdu_header->isInitialized() == false) {
         m_cmdu_header = nullptr;
+        return false;
+    }
+    
+    if (!parse_tlvs)
+        return true;
+
+    while (auto tlv = parseNextTlv()) {
```

Comment:
If there is an error and tlvEndofMessage is not exist, we could get stuck here on an infinite loop. I think you should add stop condition with a maximum number of iteration we would accept, and return false if we reach it.

Reply:
Not needed, we will fail on parseNextTlv() which will reach the end of the buffer. No infinite loop here, error code will be handled internally.